### PR TITLE
py-rtree: set the library path also for spack modules depending on py-rtree [NSE-1616]

### PR DESCRIPTION
**Pull request by lguyot**
This change sets the environment variables `SPATIALINDEX_LIBRARY` and `SPATIALINDEX_C_LIBRARY` for the spack packages which depends on `py-rtree`.

This solves in particular the first issue reported here: https://bbpteam.epfl.ch/project/issues/browse/NSETM-1616

### CODE REVIEW

**Review by matz-e — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-rtree/package.py [hunk 1]
```diff
@@ -25,3 +25,6 @@ def setup_build_environment(self, env):
 
     def setup_run_environment(self, env):
         self.setup_build_environment(env)
+
+    def setup_dependent_run_environment(self, env, dependent_spec):
+        self.setup_build_environment(env)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-rtree/package.py]:
```suggestion
        self.setup_build_environment(env)
```

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-rtree/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-rtree/package.py]: Quickly fixed that myself… in general, files should end with a newline, which some editors don't do :-/

Comment (lguyot) [reply var/spack/repos/builtin/packages/py-rtree/package.py]: @matz-e Thanks for the fix. I usually run flake8 before submitting a PR, but this time I was obviously sloppy.

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-rtree/package.py]: It was a simple change :)

**Pull request merged**